support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx

```diff
@@ -619,36 +591,13 @@ export function ThreeTierLanding({
 						/>
 					)}
 					<h1 css={heading}>
-						{getHeadline(
-							showCountdown,
-							currentCountdownSettings,
-							campaignSettings,
-							countryGroupId,
-						)}
+						<span dangerouslySetInnerHTML={{ __html: settings.copy.heading }} />
```

Comment:
Such a great name! Why is it dangerous?  😂

Reply:
Yes it's really not ideal!
This lets us pass in a string containing html, for rendering. For e.g. bold text. Although it does prevent <script> tags, there are other ways of injecting bad code into a page this way. So we need to be very confident that we control whatever is in the string.
I noticed we do [already use it elsewhere](https://github.com/guardian/support-frontend/blob/e3b1f322cb06e8b218bc5c809c4c44ea2b432686/support-frontend/assets/helpers/productPrice/promotions.tsx#L146) on the support site for rendering copy from the promos tool.
But perhaps we should be more careful now! One approach is to sanitize using e.g. https://github.com/cure53/DOMPurify.

Reply:
And in fact we already have dompurify in this project!
I've [pushed a commit](https://github.com/guardian/support-frontend/pull/6655/commits/b0ef219766c97bfe413e96554b1aeea8f6ad758e) to use it to sanitise the strings before using them

Reply:
Thanks for the comment!

Reply:
Excellent!